lib/js/components/Banner/Banner.vue

```diff
@@ -0,0 +1,375 @@
+<template>
+	<div class="o-banner" :class="[colorClass, layoutClass]">
+		<div class="o-banner__content">
+			<div class="o-banner__header">
+				<div v-if="icon" class="o-banner__iconContainer">
+					<!-- TODO: https://bethink.atlassian.net/browse/IT-3589 change to a-illustration in the future -->
+					<ds-icon class="o-banner__icon" :class="[colorClass]" :icon="icon" />
+				</div>
+
+				<div class="o-banner__text">
+					<div class="o-banner__title" v-text="title" />
+					<div v-if="hasDefaultText" class="o-banner__defaultText">
+						<slot name="defaultText" />
+					</div>
+					<div v-if="buttonText.length > 0" class="o-banner__buttonTextVertical">
+						<ds-button
+							:color="BUTTON_COLORS.NEUTRAL"
+							:type="BUTTON_TYPES.OUTLINED"
+							:size="BUTTON_SIZES.SMALL"
+							@click.native="$emit('button-clicked')"
+							>{{ buttonText }}
+						</ds-button>
+					</div>
+				</div>
+				<div v-if="buttonText.length > 0" class="o-banner__buttonText">
+					<ds-button
+						:color="BUTTON_COLORS.NEUTRAL"
+						:type="BUTTON_TYPES.OUTLINED"
+						:size="BUTTON_SIZES.SMALL"
+						@click.native="$emit('button-clicked')"
+						>{{ buttonText }}
+					</ds-button>
+				</div>
+			</div>
+			<div v-if="hasExpandedText" class="o-banner__expander">
+				<ds-icon-button
+					:size="ICON_BUTTON_SIZES.SMALL"
+					:icon="expanded ? ICONS.FA_CHEVRON_UP : ICONS.FA_CHEVRON_DOWN"
+					:color="ICON_BUTTON_COLORS.NEUTRAL"
+					:radius="BUTTON_RADIUSES.ROUNDED"
+					:touchable="false"
```

Comment:
I'm not sure if it's a good idea. Let's see what UX says in https://bethinkteam.slack.com/archives/CFBLF5RGA/p1657180642502409

Reply:
It's fine for now 🤷‍♂️